### ISSUE TITLE
Windows 10 "Could not find bibliography file"

### ISSUE THREAD

**Issue by Jaifroid** [jgm/pandoc#6291]
I just tried to update an old copy of pandoc (2.4) to the latest (2.9.2.1), and it broke the ability of pandoc to find my bibliography file. The syntax that works perfectly under 2.4 is:

`--bibliography='C:\Users\xxxxx\Books\My Library.json'`

When run under 2.9.2.1 the error "could not find bibliography file is displayed". I also tried installing 2.7.3, and ran into the same error, so it seems this bug has been in pandoc for a while.

I had to revert to 2.4 for now.

**Comment by jgm** [comment]
Does the error message give the path it's looking for or any other information?

**Comment by Jaifroid** [comment]
@jgm The error it reports is:

> Could not find bibliography file: 'C:\Users\xxxxx\Books\My Library.json'
> Error running filter pandoc-citeproc:
> Filter returned error status 1

**Comment by jgm** [comment]
That error message indicates, I think, that pandoc-citeproc is considering the single quotes as part of the filename.  I don't know much about Windows.  Do single quotes really work in this context?  Try with double quotes?

**Comment by Jaifroid** [comment]
Thank you @jgm! Using double quotes fixes it. Normally Windows / PowerShell scripts are indifferent to use of single or double, but they are needed because of the space in the filename.

So that I haven't completely wasted your time with a noob query, I have noticed that if I set

> --resource-path="C:\Users\xxxxx\Documents\Research Projects\Edited Books\Espacios necropolíticos\"

... then pandoc 2.9..2.1 complains it cannot find the resources. However, if I set it like this:

> --resource-path="C:\\Users\\xxxxx\\Documents\\Research Projects\\Edited Books\\Espacios necropolíticos\\"

... then it finds the resources there (images linked in the markdown document I am converting to docx).

I realize this is a different issue, and I'm happy to open it and test different combinations if you think it is an issue. Meanwhile, I close this one.
